src/molgenis/capice/validators/input_validator.py

```diff
@@ -8,7 +8,7 @@ class InputValidator:
     Validator for the CLI arguments
     """
     @staticmethod
-    def validate_input_path(input_path, extension: tuple):
+    def validate_input_path(input_path, extension: str | tuple[str]):
```

Comment:
Perhaps also define a type for `input_path` (I beleive `os.PathLike` can be used)

Reply:
Added `_typeshed.StrOrBytesPath` as `input_path` is I think still Strings when given to that method? Might change it to `os.PathLike` in json PR as I need to make some IO changes anyway so might just include conversion to path or so.

Reply:
Strike the above, `_typeshed.StrOrBytesPath` is not available during runtime so building failed.